src/tooltipServiceWrapper.ts

```diff
@@ -97,53 +62,57 @@ class TooltipServiceWrapper implements ITooltipServiceWrapper {
         // Mouse events
         selection.on("mouseover.tooltip", () => {
             // Ignore mouseover while handling touch events
-            if (!this.canDisplayTooltip(event)) {
+            if (!this.canDisplayTooltip(d3Event))
                 return;
-            }
 
             let tooltipEventArgs = this.makeTooltipEventArgs<T>(rootNode, true, false);
-            if (!tooltipEventArgs) { return; }
+            if (!tooltipEventArgs)
+                return;
 
             let tooltipInfo = getTooltipInfoDelegate(tooltipEventArgs);
-            if (tooltipInfo == null) { return; }
+            if (tooltipInfo == null)
+                return;
 
             let selectionId = getDataPointIdentity(tooltipEventArgs);
 
             this.visualHostTooltipService.show({
                 coordinates: tooltipEventArgs.coordinates,
+                isTouchEvent: false,
                 dataItems: tooltipInfo,
                 identities: selectionId ? [selectionId] : [],
-                isTouchEvent: false,
             });
         });
 
         selection.on("mouseout.tooltip", () => {
             this.visualHostTooltipService.hide({
-                immediately: false,
                 isTouchEvent: false,
+                immediately: false,
             });
         });
 
         selection.on("mousemove.tooltip", () => {
             // Ignore mousemove while handling touch events
-            if (!this.canDisplayTooltip(event)) { return; }
+            if (!this.canDisplayTooltip(d3Event))
+                return;
```

Comment:
This is not recommend, please always use curly brackets

Reply:
fixed